### ISSUE TITLE
Ghostnet training details

### ISSUE THREAD

**Issue by leondgarse** [cavalleria/cavaface.pytorch#77]
Hi,
I'm trying to reproduce your `GhostNet` accuracy in my [keras project](https://github.com/leondgarse/Keras_insightface). But the result is not as good... Can you share me your training details?
- In the default `config.py`, learning rate is configured as `LR_SCHEDULER = 'cosine'`, `NUM_EPOCH = 16`, is this how `Ghostnet` been trained?
- How have you cleaned the ms1m-retinaface dataset?
- Or if any other parameters set different from default `config.py`?

**Comment by cavalleria** [comment]
yes. i modify output setting of ghostnet to GDC, you can try it.

**Comment by leondgarse** [comment]
Ya, I'm using `GDC`, and compared the model structure with your shared `GhostNet_x1.3` to make them match.
I just ran some tests on ` ms1m-retinaface` using learning rate similar as:
```py
BATCH_SIZE = 512,
LR = 0.1,
LR_SCHEDULER = 'step',
LR_DECAY_EPOCH = [20, 30, 40],
NUM_EPOCH = 50,
```
But the result is ... poor...

**Comment by cavalleria** [comment]
You use relu as activation in your network. just try using PReLU.

**Comment by leondgarse** [comment]
Ya, I'm using `relu` in ghostnet model define, and another function `replace_ReLU_with_PReLU` to replace it with `PReLU`. As `PReLU` is a bit slower than `relu`. I have a test result here [TFLite model inference time test on ARM64](https://github.com/leondgarse/Keras_insightface#tflite-model-inference-time-test-on-arm64).
In training I'm actually using `PReLU`.
```py
basic_model = models.replace_ReLU_with_PReLU(basic_model)
```

**Comment by cavalleria** [comment]
sorry. i have no idea.

**Comment by leondgarse** [comment]
Ok, then I need to find it myself.
Thanks. :)
